analyzers/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldContainAssertion.NUnit.cs

```diff
@@ -137,6 +137,12 @@ public void TestMethod16()
         public void Test17() // Don't raise on skipped test methods
         {
         }
+
+        [Test(ExpectedResult = 69)]
```

Comment:
Could you please change the expected result to something more conventional :)
One option would be to use "the answer to life the universe and everything".

Reply:
Done. But, offended by a number? And 69 has that wonderful symmetry.

Reply:
Not for me personally, but we should keep this safe for everybody.